hwangsae/transmuxer.c

```diff
@@ -300,6 +300,22 @@ hwangsae_transmuxer_clear (HwangsaeTransmuxer * self)
   gst_clear_object (&priv->pipeline);
 }
 
+static GSList *
+_check_input_files (GSList * input_files)
+{
+  for (GSList * it = input_files; it;) {
+    GSList *next = g_slist_next (it);
+    gchar *file = it->data;
+    if (!g_file_test (file, G_FILE_TEST_IS_REGULAR)) {
+      g_warning ("File %s not found, ominting it", file);
```

Comment:
typo in omitting

Reply:
I'm not sure warning is enough here as it may get overlooked. Having file that doesn't exist in the list can mean a mistake in user input and I'd lean more towards returning an error from `hwangsae_transmuxer_merge ()` and doing nothing. Better than to make an incomplete mux due to a simple typo in a shell command.

Reply:
I see your point and I also wonder how should we handle this, which should depends on application . For a command line app I agree with you, for a service not sure.
As the project is about to end and the current application is a command line tool, let's raise an error.